website/docs/reference/warehouse-profiles/databricks-profile.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Databricks Profile"
+id: "databricks-profile"
+---
+
+## Overview of dbt-spark
+
+**Maintained by:** some dbt loving bricksters
+**Author:** Databricks
+**Source:** [Github](https://github.com/databricks/dbt-databricks)    
+**dbt Cloud:** Coming Soon    
+**dbt Slack channel** [Link to channel](https://getdbt.slack.com/archives/CNGCW8HKL)     
+
+
+![dbt-databricks stars](https://img.shields.io/github/stars/databricks/dbt-databricks?style=for-the-badge)
+
+## Installation and Distribution
+
+The easiest way to install dbt-databricks is to use `pip`:
+
+```zsh
+pip install dbt-databricks
+```
+
+### Set up a Databricks Target
+
+dbt-databricks can connect to a Databricks all-purpose cluster as well as a SQL endpoints.
```

Comment:
This is something that I noticed trips customers up a lot. Could you point to any documentation/write a blurb to help clients understand the pros and cons?